code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -1289,7 +1289,10 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 	var/husk = (HUSK in mutations)
 	var/hulk = (HULK in mutations)
 	var/skeleton = (SKELETON in mutations)
-
+	var/g = dna.GetUITriState(DNA_UI_GENDER)
```

Comment:
This will mean that every time this proc is called that the human will look different

Reply:
:clique: :fingerguns: >__<;